tests/zfs-tests/tests/functional/cli_root/zpool_create/create-o_ashift.ksh

```diff
@@ -62,7 +62,13 @@ function txg_sync
 
 	while [ $i -lt $count ]
 	do
-		log_must sync_pool $pool true
+#		FIXME: log_must sync_pool $pool true
+		zfs snapshot $pool@sync$$$i
+		if [[ $? -ne 0 ]]
+		then
+			log_fail "Failed to sync pool $pool (snapshot $i)"
+			return 1
+		fi
```

Comment:
Since i just reproduced this same failure on a local `ubuntu-14.04.4-server-i386` builder i'm dropping this change from this PR and will continue working on this "offline"; i'm going to open a new PR if/when i'm able to fix the underlying issue.